src/nise/metadata/config.py

```diff
@@ -19,6 +19,7 @@
     },
     'context_manager_name': {},
     'init_function': 'OpenSession',
+    'close_function': 'CloseSession',
```

Comment:
If `init_function` is not optional, I don't think `close_function` should be optional.

Reply:
Changed